projects/hip-tests/catch/README.md

```diff
@@ -169,14 +169,73 @@ REQUIRE(exepctedOutput == proc.getOutput()); // Test on expected output of the p
 ```
 The process must be a standalone exe inside the same folder as other tests.
 
-## Enabling New Tests
-Initially, the new tests can be enabled via using ```-DHIP_CATCH_TEST=1```. After porting existing tests, this will be turned on by default.
-
 ## Building a single test
+
+The  test suite has moved to Catch2v3 (v3.8.1) to be exact. Moving from v2 to v3 came with some fundamental changes in how Catch2 interacts with hip-tests.
+Starting with, it is no longer a single header, it is now a library, which needs to be linked to the test exe.
+
+If you are on your personal machine, I highly recommend you to install Catch2v3 (v3.8.1) locally on your system. This helps skip the download/build part in hip-tests and results in faster builds overall.
+
+### Steps to install Catch2v3 locally
+
+- `git clone https://github.com/catchorg/Catch2.git -b v3.8.1 --depth 1`
+- `cd Catch2`
+- `mkdir build && cd build`
+- `cmake .. -DCMAKE_BUILD_TYPE=Release`
+- `make -j8`
+
+Install step, you might need to have superuser permission to be able to install globally.
+
+- `make install`
+
+With Catch2 installed globally we can build individual Catch2 test like this:
+
 ```bash
-hipcc <path_to_test.cpp> -I<HIP_SRC_DIR>/tests/catch/include <HIP_SRC_DIR>/tests/catch/hipTestMain/standalone_main.cc -I<HIP_SRC_DIR>/tests/catch/external/Catch2 -g -o <out_file_name>
+export ROCM_SYSTEMS_DIR=<path_to_rocm_systems>
```

Comment:
I need to try on windows to see if these work.  But I can fix this Readme later with updated instructions